packages/web-twig/src/Resources/components/UNSTABLE_EmptyState/UNSTABLE_EmptyState.twig

```diff
@@ -0,0 +1,83 @@
+{# API #}
+{%- set props = props | default([]) -%}
+{%- set _buttonPrimaryText = props.buttonPrimaryText | default(null) -%}
+{%- set _buttonPrimaryUrl = props.buttonPrimaryUrl | default(null) -%}
+{%- set _buttonSecondaryText = props.buttonSecondaryText | default(null) -%}
+{%- set _buttonSecondaryUrl = props.buttonSecondaryUrl | default(null) -%}
+{%- set _description = props.description -%}
+{%- set _headingElementType = props.headingElementType | default('h2') -%}
+{%- set _iconName = props.iconName | default(null) -%}
+{%- set _linkText = props.linkText | default(null) -%}
+{%- set _linkUrl = props.linkUrl | default(null) -%}
+{%- set _title = props.title -%}
```

Comment:
This will clash with the HTML `title` attribute. Do we change it now or later?

Reply:
Lately, it will probably be removed as we discussed the composition.